crates/trie/sparse/src/lib.rs

```diff
@@ -1,6 +1,24 @@
 //! The implementation of sparse MPT.
 
 #![cfg_attr(not(test), warn(unused_crate_dependencies))]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(not(feature = "std"))]
```

Comment:
we can add this but cleaner would be adding extern crate alloc; and update all imports

Reply:
Ah okay so just using alloc instead of std, will change